.changeset/mean-poets-drum.md

```diff
@@ -0,0 +1,22 @@
+---
+'@scaleway/regex': major
+---
+
+Rename few regex and camelcase all of them for more consistency. Also creates alphanumDashUnderscoreDots.
+
+Here is the full list of changes:
+
+- alphaLower -> alphaLowercase
+- alphanumdash -> alphanumDash
```

Comment:
`alphanum` should be `alphaNum` I think?

Reply:
alphanumeric is a valid word so no need for camelCase